src/notebook/plugin.ts

```diff
@@ -152,7 +152,14 @@ function activateCellTools(app: JupyterLab, restorer: IInstanceRestorer, tracker
   celltools.addItem({ tool: metadataEditor, rank: 4 });
 
   restorer.add(celltools, namespace);
-  app.shell.addToLeftArea(celltools);
```

Comment:
Even if the first Panel that was created is a notebook panel, the signal is sent. So, there is no necessity for it as it will be caught when the signal raises it

I tested this by opening a notebook and refreshing the page - JupyterLab saves the fact that the notebook was the panel I was at and reopens only the notebook. In this case also the CellToolbar is shown